### ISSUE TITLE
Get Height of Row

### ISSUE THREAD

**Issue by TheSilentHorizon** [vandeseer/easytable#26]
Hey, sorry for no response on the other thread, will get to that later. So, question:

row.getHeight() only works apparently when i fake a table like here:

```
   private float getRowHeight(Row row) {
        Table dummy = baseTable().addRow(row).build();
        return row.getHeight();
    }
```

Is there a way to return the expected row Height (need it for formatting multipage bills)

**Comment by TheSilentHorizon** [comment]
Ok, figured it out myself:

To calculate the RowHeight, the Row needs to have all information later added from the table,

So something like this works:

`        Table dummy = Table.builder().addColumnOfWidth(70)
                .addColumnOfWidth(270)
                .addColumnOfWidth(70)
                .addColumnOfWidth(70).wordBreak(true).addRow(row).build();`

I dont like this solution but it aint that messy.

**Comment by vandeseer** [comment]
Hey @TheSilentHorizon, please also have a look at the discussion here: https://github.com/vandeseer/easytable/issues/23 – in particular this comment: https://github.com/vandeseer/easytable/issues/23#issuecomment-445550423.

TLDR: If you need to build a table over several pages there is feature for that ;)
